src/database/typeDefs.graphql

```diff
@@ -26,21 +26,11 @@ type Source {
 	logoLink: String
 }
 
-type TwitterHandle {
-	_id: String!
-	name: String
-	handle: String!
-	category: String
-	userWeight: String
-	categoryWeight: String
-}
-
 type Tweet {
 	_id: String!
 	text: String
 	tweetId: String
 	handle: String
-	twitterHandle: TwitterHandle
```

Comment:
@ShivaAryal Deleting this caused backward-compatibility issue as old app was expecting this field to be present